apps/runtime-demo/3005-runtime-host/runtimePlugin.ts

```diff
@@ -0,0 +1,16 @@
+import { FederationRuntimePlugin } from '@module-federation/runtime/types';
+
+export default function (): FederationRuntimePlugin {
+  return {
+    name: 'custom-plugin-build',
+    beforeInit(args) {
```

Comment:
I think we need to integrate testing with test-related functionality

Reply:
Absolutely, especially since these hooks are evolving